index.html

```diff
@@ -45,6 +45,183 @@ <h1 class="headline">
+                        <li class="github"><a href="https://github.com/ishpaul777"><img src="./assets/icons/github.svg" alt="github"></a></li>
+                        <li class="linkedin"><a href="https://www.linkedin.com/in/ishpaul-singh-264590226/"><img src="./assets/icons/linkedin.svg" alt="linkedin"></a></li>
+                        <li class="twitter"><a href="https://twitter.com/ishpaul_777"><img src="./assets/icons/twitter.svg" alt="twitter"></a></li>
+                        <li class="instagram"><a href="https://www.instagram.com/ishpaul_777"><img src="./assets/icons/instagram.svg" alt="instagram"></a></li>
+                    </ul>
+                </div>
+                <button type="button" class="resume-btn">Get my resume</button>
+                <div class="skills-info">
+                    
+                    <ul class="languages">
+                        <li class="skill-type">
+                            <p>languages</p>
+                            <a href="#"><img src="/assets/icons/chevron-down.svg" alt="down-arrow"></a>
+                        </li>
+                    </ul> 
+                    <ul class="lang-skill">
+                        <li>
+                            <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript" class="lang-type">
+                                <img src="/assets/icons/javascript-logo.png" alt="down-arrow">
+                                <p>Javascript</p>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://developer.mozilla.org/en-US/docs/Web/HTML" class="lang-type">
+                                <img src="/assets/icons/HTML-logo.png" alt="down-arrow">
+                                <p>HTML</p>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://developer.mozilla.org/en-US/docs/Web/CSS" class="lang-type">
+                                <img src="/assets/icons/css-logo.png" alt="down-arrow">
+                                <p>CSS</p>
+                            </a>
+                        </li>
+                    </ul>
+                    
+                    <ul class="frameworks">
+                        <li class="skill-type">
+                            <p>Frameworks</p>
+                            <a href="#"><img src="/assets/icons/chevron-right.svg" alt="right-arrow"></a>
+                        </li>
+                    </ul>
+                    <ul class="languages">
+                        <li class="skill-type">
+                            <p>Skills</p>
+                            <a href="#"><img src="/assets/icons/chevron-right.svg" alt="right-arrow"></a>
+                        </li>
+                    </ul>
+                    
+                            
+                    
+                </div>
+            </section>
```

Comment:
As you can see, this section of your application is breaking. There might be some issue with the icons you are using. Please fix it. 😊
![issue](https://user-images.githubusercontent.com/33268927/170674870-07504805-e670-4b55-8d6b-87f7e2eabd45.png)